src/endToEnd.test.ts

```diff
@@ -2,10 +2,10 @@ import { expect } from 'chai';
 import IPFS from 'ipfs';
 import StellarSdk from 'stellar-sdk';
 
-import RahaStellar, { getNewTestAccount } from "./RahaStellar";
-import { saveDataToIpfsAsFile } from "./RahaIpfs";
-import { getBlockchain } from './RahaBlockchain';
-import { IpfsBlock } from './RahaSchema';
+import Stellar, { getNewTestAccount } from "./modules/stellar";
+import { saveDataToIpfsAsFile } from "./modules/ipfs";
+import { getBlockchain } from './modules/blockchain';
```

Comment:
Nomenclature, but module is just a really general term in node - it literally refers to any file that exports anything. I’d be fine with you just not even putting it in a sub directory unless there’s a more specific grouping here